src/sentry/lang/native/minidump.py

```diff
@@ -16,33 +13,6 @@
 MAX_MSGPACK_EVENT_SIZE_BYTES = 100000
```

Comment:
All these constants are dead now, including the logger.

The only exception is `MINIDUMP_ATTACHMENT_TYPE` which is only used in `.native.processing`. Please move that over.